src/main/frontend/core.cljs

```diff
@@ -13,7 +14,8 @@
             [reitit.frontend.easy :as rfe]
             [logseq.api]
             [frontend.fs.sync :as sync]
-            [frontend.config :as config]))
+            [frontend.config :as config]
+            [malli.dev.cljs :as md]))
```

Comment:
I don't think this dev ns will grow the prod asset too much but if it does it looks like we could address it quick - https://github.com/metosin/malli/blob/master/docs/clojurescript-function-instrumentation.md#releaes-builds